homework/27-28/week-01/README.md

```diff
@@ -48,10 +48,8 @@ Please use any remaining time to complete and review the exercises from today:
 <!--
 ## Day 4
 
-1. Reading
-2. Friday Review Prep
-    - Complete the [Week 1 Self-Assessment](#PENDING) and identify 2 topics you want to review tomorrow
-    - Ask and/or upvote 3 questions on QuestionCookie: http://www.questioncookie.com/wdi-27-28-w1-review
```

Comment:
I need to migrate the assessment questions from the spreadsheet to form. This is rather laborious as it has to be done by hand.